src/parser.rs

```diff
@@ -332,7 +334,8 @@ fn parse_message_s(m: &str) -> ParseResult<SyslogMessage> {
        severity: sev,
        facility: fac,
        version,
-       timestamp,
+       timestamp: event_time.map(|t| t.sec),
+       timestamp_millis: event_time.map(|t| t.nsec / NSEC_PER_MS),
```

Comment:
I'm not sure I love having a division in the critical path here; might be better to just natively expose nanos